src/rard/research/views/antiquarian.py

```diff
@@ -77,10 +78,10 @@ def post(self, *args, **kwargs):
             try:
                 model_class = model_classes[object_type]
                 link = model_class.objects.get(pk=link_pk)
-                if "up_by_work" in self.request.POST:
-                    link.move_to_by_work(link.work_order - 1)
-                elif "down_by_work" in self.request.POST:
-                    link.move_to_by_work(link.work_order + 1)
+                if "up_by_book" in self.request.POST:
+                    link.move_to_by_book(link.work_order - 1)
+                elif "down_by_book" in self.request.POST:
+                    link.move_to_by_book(link.work_order + 1)
```

Comment:
It's order_in_book +/- 1 we want to pass as the new position here I think:
```suggestion
                if "up_by_book" in self.request.POST:
                    link.move_to_by_book(link.order_in_book - 1)
                elif "down_by_book" in self.request.POST:
                    link.move_to_by_book(link.order_in_book + 1)
```